examples/pcf8563/time/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/pcf8563"
+)
+
+var (
+	i2c = machine.I2C0
+	rtc = pcf8563.New(i2c)
+)
+
+func main() {
+	i2c.Configure(machine.I2CConfig{Frequency: machine.TWI_FREQ_400KHZ})
+
```

Comment:
Is this missing a call to `rtc.Reset()` here?

Reply:
Here, `Reset()` may or may not be called, but this example may make some people wonder.
I will add a call to the `Reset()` function here, and also add some more information in the comments of the `Reset()` function.